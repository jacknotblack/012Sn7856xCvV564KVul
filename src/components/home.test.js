import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Home from "./home";

describe("A suite", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Home />);
  });
  it("renders without crashing", () => {
    expect(wrapper.length).toEqual(1);
  });

  it("should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
