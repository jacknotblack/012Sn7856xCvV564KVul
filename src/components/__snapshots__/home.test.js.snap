// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite should match snapshot 1`] = `
<div>
  Home 
  <br />
  <ul>
    <li>
      <Link
        replace={false}
        to="/ranks"
      >
        Ranks
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/billing"
      >
        Billing
      </Link>
    </li>
  </ul>
</div>
`;
